<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfirmQSO</class>
 <widget class="QDialog" name="ConfirmQSO">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Подтверждение QSO</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="QSOCountLabel">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Всего подтвержденных QSO:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="CurrentQSOCountLabel">
          <property name="text">
           <string>Загружено подтвержденных QSO:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="ConfirmProgressBar">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Параметры загрузки</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Дата начала:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateEdit">
          <property name="minimumDateTime">
           <datetime>
            <hour>0</hour>
            <minute>0</minute>
            <second>0</second>
            <year>2024</year>
            <month>10</month>
            <day>24</day>
           </datetime>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Позывной станции:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="CallSignComboBox"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="insertCheckBox">
        <property name="text">
         <string>Добавлять отсутствующие в базе связи</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ConfirmButton">
       <property name="text">
        <string>Подтвердить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CloseCloseButton">
       <property name="text">
        <string>Закрыть</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>dateEdit</tabstop>
  <tabstop>CallSignComboBox</tabstop>
  <tabstop>insertCheckBox</tabstop>
  <tabstop>ConfirmButton</tabstop>
  <tabstop>CloseCloseButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
